components/fctl/cmd/payments/connectors/install/atlar.go

```diff
@@ -0,0 +1,97 @@
+package install
+
+import (
+	"fmt"
+
+	"github.com/formancehq/fctl/cmd/payments/connectors/internal"
+	fctl "github.com/formancehq/fctl/pkg"
+	"github.com/formancehq/formance-sdk-go/pkg/models/operations"
+	"github.com/formancehq/formance-sdk-go/pkg/models/shared"
+	"github.com/pkg/errors"
+	"github.com/pterm/pterm"
+	"github.com/spf13/cobra"
+)
+
+type PaymentsConnectorsAtlarStore struct {
+	Success       bool   `json:"success"`
+	ConnectorName string `json:"connectorName"`
+}
+
+type PaymentsConnectorsAtlarController struct {
+	store              *PaymentsConnectorsAtlarStore
+	atlarAccessKeyFlag string
+	atlarSecretFlag    string
+}
+
+var _ fctl.Controller[*PaymentsConnectorsAtlarStore] = (*PaymentsConnectorsAtlarController)(nil)
+
+func NewDefaultPaymentsConnectorsAtlarStore() *PaymentsConnectorsAtlarStore {
+	return &PaymentsConnectorsAtlarStore{
+		Success: false,
+	}
+}
+
+func NewPaymentsConnectorsAtlarController() *PaymentsConnectorsAtlarController {
+	return &PaymentsConnectorsAtlarController{
+		store:              NewDefaultPaymentsConnectorsAtlarStore(),
+		atlarAccessKeyFlag: "access-key",
+		atlarSecretFlag:    "secret",
+	}
+}
+
+func NewAtlarCommand() *cobra.Command {
+	c := NewPaymentsConnectorsAtlarController()
+	return fctl.NewCommand(internal.AtlarConnector+" <access-key> <secret>",
+		fctl.WithShortDescription("Install an atlar connector"),
+		fctl.WithConfirmFlag(),
+		fctl.WithArgs(cobra.ExactArgs(2)),
+		fctl.WithStringFlag(c.atlarAccessKeyFlag, "", "Atlar API access key"),
+		fctl.WithStringFlag(c.atlarSecretFlag, "", "Atlar API secret"),
+		fctl.WithController[*PaymentsConnectorsAtlarStore](c),
+	)
+}
+
+func (c *PaymentsConnectorsAtlarController) GetStore() *PaymentsConnectorsAtlarStore {
+	return c.store
+}
+
+func (c *PaymentsConnectorsAtlarController) Run(cmd *cobra.Command, args []string) (fctl.Renderable, error) {
+	soc, err := fctl.GetStackOrganizationConfigApprobation(cmd, "You are about to install connector '%s'", internal.AtlarConnector)
+	if err != nil {
+		return nil, fctl.ErrMissingApproval
+	}
+
+	paymentsClient, err := fctl.NewStackClient(cmd, soc.Config, soc.Stack)
+	if err != nil {
+		return nil, err
+	}
+
+	response, err := paymentsClient.Payments.InstallConnector(cmd.Context(), operations.InstallConnectorRequest{
+		ConnectorConfig: shared.ConnectorConfig{
+			AtlarConfig: &shared.AtlarConfig{
```

Comment:
AtlarConfig does not exist yet, you have to add it inside the openapi.yaml in the components/payments directory.

If you need any help on this matter, don't hesitate to ping me, I will be glad to help you !

Reply:
Thanks for pointing that out! I assumed that would be generated. I'll add it and come back to you in case I run into problems.

I really appreciate you taking the time to take a look over this PR 🚀

Reply:
I added the missing definition of the connector config. Thanks again for pointing it out 🙌

Reply:
Thank you !